presto-iceberg/src/main/java/com/facebook/presto/iceberg/FilesTable.java

```diff
@@ -146,15 +146,15 @@ private static List<Page> buildPages(ConnectorTableMetadata tableMetadata, Conne
                         .filter(entry -> idToTypeMap.containsKey(entry.getKey()))
                         .collect(toImmutableMap(
                                 Map.Entry<Integer, ByteBuffer>::getKey,
-                                entry -> Transforms.identity(idToTypeMap.get(entry.getKey())).toHumanString(
+                                entry -> Transforms.identity().toHumanString(idToTypeMap.get(entry.getKey()),
```

Comment:
`Transforms.identity(<type>)` has been deprecated and will be removed in 2.0.0, so we can change it now to lazy bind to the given type